gateway-service-impl/src/main/java/org/hypertrace/gateway/service/common/converters/StringToAttributeKindConverter.java

```diff
@@ -92,6 +94,12 @@ private List<String> convertToArray(String jsonString) {
       return List.of();
     }
 
+    // handle special case when "null" string is returned as string array value(default value
```

Comment:
Discussed offline - concerned about this behavior change, as every query that was previously returning an empty array (e.g. give me the labels for this API) will now start containing nulls. Existing behavior seemed more accurate, we just need to make sure we appropriately handle group by case and either return no groups (short circuit) or translate it back if we think we should look for explicitly empty groups as part of a group by query. I'm not sure what the actual correct expectation should be there.